framework/include/security/security_common.h

```diff
@@ -21,10 +21,10 @@
 
 #include <stdint.h>
 
-#define SECURITY_MAX_BUF              1024
 #define SECURITY_MAX_KEY_BUF          256
 #define SECURITY_MAX_CERT_BUF         2048
 #define SECURITY_MAX_SS_BUF           4096
+#define SECURITY_MAX_BUF              SECURITY_MAX_SS_BUF
```

Comment:
Where SECURITY_MAX_SS_BUF is defined..? Is SECURITY_MAX_SS_BUF 1024?

Reply:
line 26 it's defined.